### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by beliaev-maksim**
without env var get

```
> actions-operator@0.1.0 build
> ncc build src/bootstrap/index.ts -o dist/bootstrap --license licenses.txt && ncc build src/cleanup/index.ts -o dist/cleanup --license licenses.txt

ncc: Version 0.27.0
ncc: Compiling file index.js
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
    at hashOf (/home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:3253134)
    at ncc (/home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:1:3256648)
    at runCmd (/home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:51537)
    at 819 (/home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:48344)
    at __webpack_require__ (/home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:55043)
    at /home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:55194
    at /home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:55220
    at Object.<anonymous> (/home/maksim/git/actions-operator/node_modules/@vercel/ncc/dist/ncc/cli.js:8:28) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Building using npm:
 
 ```
 sudo apt install npm  # install npm, if needed
-npm install           # install / update dependencies
+export NODE_OPTIONS=--openssl-legacy-provider
+npm install           # install dependencies
 npm run build
 ```
+
```
Comment (addyess) [inline CONTRIBUTING.md:37]: TBH -- i have just been using the node snap.  I guess this is right if you're using apt??

Comment (addyess) [reply CONTRIBUTING.md:37]: @ca-scribner any thoughts on updating this?

Comment (beliaev-maksim) [reply CONTRIBUTING.md:37]: I am also on snap :D 
image

Comment (beliaev-maksim) [reply CONTRIBUTING.md:37]: @addyess 
I think my NPM comes from node snap.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ Building using npm:
 
 ```
 sudo apt install npm  # install npm, if needed
-npm install           # install / update dependencies
+export NODE_OPTIONS=--openssl-legacy-provider
+npm install           # install dependencies
```
Comment (addyess) [inline CONTRIBUTING.md:34]: so how about:

```
sudo snap install node --channel=18/stable
```

Comment (beliaev-maksim) [reply CONTRIBUTING.md:34]: that makes sense. However, I assume we still will need to export this variable

File: CONTRIBUTING.md (hunk 1 above)
Comment (ca-scribner) [reply CONTRIBUTING.md:37]: main question I have is if things were working before, why did they stop?

I see [this SO thread](https://stackoverflow.com/questions/72866798/node-openssl-legacy-provider-is-not-allowed-in-node-options) that sounds like doing `export NODE_OPTIONS=--openssl-legacy-provider` is the *cause* of problems.  I don't know anything about node or npm, but that makes me wonder

Comment (ca-scribner) [reply CONTRIBUTING.md:37]: so yeah if there's a way that avoids the export, maybe that's safer.  

it is just developer notes so not like it'll be a disaster if it doesn't fit everyone, but still

Comment (beliaev-maksim) [reply CONTRIBUTING.md:37]: @ca-scribner actually, this is opposite. This var helps to avoid package issues

Comment (ca-scribner) [reply CONTRIBUTING.md:37]: This is my point :)  It solves *your* issue, but seems to cause someone else's issue.  Without knowing why, I'm inclined to use a solution that avoids it entirely.

But again, I know nothing about node/npm, so if there's a nuance about this I wont see it

**Comment by addyess** [comment]
Closing in favor of #80

**Pull request closed without merging**